docs/bigtable.txt

```diff
@@ -1,6 +1,5 @@
 [[bigtable]]
-Google Cloud Bigtable
----------------------
+== Google Cloud Bigtable
```

Comment:
That's what I started with, but it turns out that the headings are document-wide, rather than file-wide. Perhaps if all the headings in every file were done that way it would be ok, but when I tried it the contents were all broken.